crates/core/tedge_agent/src/software_manager/error.rs

```diff
@@ -24,6 +24,9 @@ pub enum SoftwareManagerError {
 
     #[error(transparent)]
     FromConfigSetting(#[from] tedge_config::ConfigSettingError),
```

Comment:
This error is related to old tedge config API.

Reply:
removed